packages/audio/alsa-utils/package.mk

```diff
@@ -57,6 +56,9 @@ post_makeinstall_target() {
     rm -rf $INSTALL/usr/bin/$i
   done
 
+# keep alsamixer for the multimedia-tools addon
+  mv $INSTALL/usr/bin/alsamixer $ROOT/$PKG_BUILD/alsamixer/
```

Comment:
make install moves alsamixer, ie removes it from $BUILD/alsamixer
if we do this, alsamixer is lost